### PR TITLE
Update setup.py and README.md

### PR DESCRIPTION
**Pull request by nd-02110114**

### CODE REVIEW

File: scripts/install_deepchem_conda.ps1 [hunk 1]
```diff
@@ -20,23 +20,17 @@ else
...
     mdtraj `
-    pdbfixer `
-    rdkit `
-    joblib `
-    scikit-learn `
     networkx `
+    openmm `
+    pdbfixer `
     pillow `
-    pandas `
```
Comment (rbharath) [inline scripts/install_deepchem_conda.ps1:30]: Don't we still need pandas as a dependency?

File: scripts/install_deepchem_conda.sh [hunk 2]
```diff
@@ -24,21 +24,16 @@ fi
...
     mdtraj \
-    pdbfixer \
-    rdkit \
-    joblib \
-    scikit-learn \
     networkx \
+    openmm \
+    pdbfixer \
     pillow \
-    pandas \
```
Comment (rbharath) [inline scripts/install_deepchem_conda.sh:34]: Same here

File: scripts/install_deepchem_conda.ps1 (hunk 1 above)
Comment (nd-02110114) [reply scripts/install_deepchem_conda.ps1:30]: hard dependencies (pandas and scikit-learn) are installed by `install_requires` in setup.py.
I thinks `install_requires` are better for managing hard dependencies. 

If you use `install_requires`, we don't need to install hard dependencies manually. 
(Current deepchem published by pypi requires manual installations)

Currently, we should install manually like this

```
pip install pandas pillow scikit-learn==0.22 tensorflow-gpu==1.14 deepchem==2.2.1.dev54
```

But after this modification, 

```
pip install tensorflow==2.2 deepchem==2.4.0
```

**Comment by nd-02110114** [comment]
After merging this https://github.com/conda-forge/deepchem-feedstock/pull/3, I also update the installation in docs. After updating docs, this PR is ready to merge

File: scripts/colab_install.py [hunk 3]
```diff
@@ -15,29 +15,58 @@
...
+    "rdkit",
+    "-c",
+    "conda-forge",
+]
+default_packages = [
+    "rdkit",
+]
+
+
+def install(
```
Comment (nd-02110114) [inline scripts/colab_install.py:29]: I refactored the more flexible script. I will make the update tutorials in another PR.

File: scripts/install_deepchem_conda.sh [hunk 4]
```diff
@@ -18,27 +18,22 @@ then
     echo "Installing DeepChem in current env"
 else
     export envname=$1
-    conda create -y --name $envname "python>=$python_version"
+    conda create -y --name $envname python=$python_version
```
Comment (nd-02110114) [inline scripts/install_deepchem_conda.sh:21]: "python>=$python_version" install python 3.8 but many package like tensorflow doesn't support python 3.8. So, conda try to downgrade python 3.6 and it is a waste of time.

**Comment by nd-02110114** [comment]
@rbharath 
This is ready to merge!

**Pull request merged**